onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -120,18 +121,19 @@ private void poll1(Cursor c) {
         ObaArrivalInfoResponse response = ObaArrivalInfoRequest
                 .newRequest(mContext, stopId).call();
 
-        Long departMS = null;
+        // Pair of departMs and is arriving information
+        Pair<Long, Boolean> depart = null;
```

Comment:
Let's change the name of `depart`, because it could no longer be a departure.  Maybe `eta`?  Although that implies an arrival.  I'd welcome any other naming ideas.